simple-social-icons.php

```diff
@@ -520,9 +520,21 @@ function get_icon_markup( $icon, $label ) {
 		return $markup;
 	}
 
+    /**
+     * Remove option when uninstalling the plugin.
+     *
+     * @since 2.0.2
+     */
+    public static function plugin_uninstall() {
+        if ( get_option( 'widget_simple-social-icons' ) ) {
```

Comment:
Looks like we can just do `delete_option` here without the `get_option` check, because `delete_option` returns false if the option isn't found.

Reply:
Totally makes sense. Resolved in the new commit.